src/lib/accessibility/Behaviors/Tab/TabBehavior.ts

```diff
@@ -0,0 +1,17 @@
+import { Accessibility } from '../../interfaces'
+
+const TabBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    root: {
+      role: 'presentation',
+    },
+    anchor: {
+      role: 'tab',
+      'aria-selected': !!props['active'],
```

Comment:
I think you won't need double exclamation mark here, it can be just set to ``` props['active'] ``` in string

Reply:
please test this with screen readers - double exclamation marks will turn anything into true/false. What is the behavior described by ARIA best practices? Should the screen reader always read "not selected" when you navigate to a tab that is not selected? or should it just read "selected" on the tab that is selected? if latter is the preferred way, I think it should be ``props['active'] ? true : undefined``

Reply:
It was tested with both double exclamation and without. It works in both cases, reason why I added is because of ambiguity. there is also a double exclamation for SelectableListItemBehavior, `'aria-selected': !!props['active'],` . How should we go consistent?

Aria states we should have true and false as values. VoiceOver screen reader will not read 'not selected' if we have "false", it will just ignore it. ChromeVox, however, will read it as 'not selected'. Afaik we support VoiceOver only, but even with ChromeVox behaviour should be ok, it's just a different way to read it, and Aria accepts it too. https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html